content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -0,0 +1,107 @@
+
+There are many blogs to introduce and analyze Ambient in community and technology forums, and this blog will analyze the network traffic path in Istio Ambient and Sidecar modes, focusing on analyzing what's the network traffic path between services in these two modes.
+
+In order to clarify the network traffic paths, this blog will explore two concrete scenarios with corresponding diagram to make it easier to understand. These two scenarios include:
+
+- **The network path of service in Ambient mode to service in Sidecar mode**
+- **The network path of service in Sidecar mode to service in Ambient mode**
+
+_Note: The following analysis is based on Istio 1.18.2 which ambient only use iptables as the redirection mode._
+
+## Ambient sleep to Sidecar httpbin
+
+### Deployment and configuration for the first scenario
+
+- sleep is deployed in namespace foo
+    - sleep pod is scheduled to Node A
+- httpbin is deployed in namespace bar
+    - httpbin is scheduled to Node B
+- foo namespace enables ambient mode (foo namespace contains label: `istio.io/dataplane-mode=ambient`)
+- bar namespace enables sidecar injection (bar namespace contains label: `istio-injection: enabled`)
+
+According to above description, the deployment and network traffic path are showing below:
+
+{{< image width="100%"
+    link="ambient-to-sidecar.png"
+    caption="Ambient sleep to Sidecar httpbin"
+    >}}
+
+`ztunnel` will be deployed as DaemonSet in istio-system namespace if Ambient is enabled, meanwhile, the components of istio-cni and ztunnel would generate separately iptables rules and routes for both ztunnel pod and its node.
+
+All network traffic coming in/out for the pod with Ambient enabled will go through ztunnel based on the network traffic hijacking mechanism. And ztunnel will handle L4 load balance, then forward the request traffic to correct endpoints.
+
+### sleep -> ztunnel -> [ sidecar -> httpbin ] network traffic path
+
+According to above diagram, the details of network traffic path is demonstrated as below:
+
+**(1) (2) (3)**  Request traffic of service sleep is sent out from `veth` of the sleep pod, then it will be marked and forwarded to the `istioout` device in the node by following the iptables rules and route rules. The `istioout` device in the node A is a `[geneve](https://www.rfc-editor.org/rfc/rfc8926.html)` tunnel, and the other side of this tunnel is pair to `pistioout` device which is inside the ztunnel pod of node A.
+
+**(4) (5)**  When the traffic arrives `pistioout` device, it goes on to be intercepted to the interface `eth0` of ztunnel pod on port 15001 by iptables rules inside the pod, so far the request traffic goes inside the ztunnel.
```

Comment:
```suggestion
**(4) (5)**  When the traffic arrives `pistioout` device, it will be intercepted through the interface `eth0` of the ztunnel pod on port 15001 by iptables rules inside the pod.
```

Reply:
already done!